modules/certificates/main.tf

```diff
@@ -0,0 +1,104 @@
+resource "kubernetes_namespace" "cert_manager" {
+  count = var.install_cert_manager ? 1 : 0
+
+  metadata {
+    name = var.cert_manager_namespace
+  }
+}
+
+resource "helm_release" "cert_manager" {
```

Comment:
The [timeout](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#timeout-1)  here defaults to defaults to 300 seconds. Is this enough or shall we set a specific timeout?

Reply:
It was long enough for me, but we can bump it if you think that is too short.